packages/react-components/src/organisms/ResearchOutputRelatedResearchCard.tsx

```diff
@@ -0,0 +1,105 @@
+import { ComponentPropsWithRef } from 'react';
+import { css } from '@emotion/react';
+
+import { FormCard, LabeledMultiSelect } from '../molecules';
+import { noop, ResearchOutputOption } from '../utils';
+import { components } from 'react-select';
+import { perRem } from '../pixels';
+import { getIconForDocumentType } from './RecentSharedOutputs';
+import { Pill } from '../atoms';
+
+const optionStyles = (showPill: boolean) =>
+  css({
+    display: 'grid',
+    gridTemplateColumns: `min-content${showPill ? ' min-content' : ''} auto`,
+    justifyItems: 'start',
+    alignItems: 'center',
+    alignContent: 'center',
+    columnGap: '12px',
+    svg: {
+      width: '18px',
+      height: '18px',
+    },
+  });
+
+type ResearchOutputRelatedResearchProps = {
+  readonly relatedResearch: ComponentPropsWithRef<
+    typeof LabeledMultiSelect<ResearchOutputOption>
+  >['values'];
+  readonly getRelatedResearchSuggestions?: ComponentPropsWithRef<
+    typeof LabeledMultiSelect<ResearchOutputOption>
+  >['loadOptions'];
+  readonly onChangeRelatedResearch?: ComponentPropsWithRef<
+    typeof LabeledMultiSelect<ResearchOutputOption>
+  >['onChange'];
+
+  readonly isSaving: boolean;
+  isEditMode?: boolean;
+};
+
+const ResearchOutputRelatedResearchCard: React.FC<
+  ResearchOutputRelatedResearchProps
+> = ({
+  relatedResearch,
+  getRelatedResearchSuggestions = noop,
+  onChangeRelatedResearch = noop,
+  isSaving,
+  isEditMode,
+}) => (
+  <FormCard title="Are there any related outputs?">
+    <LabeledMultiSelect<ResearchOutputOption>
+      title="Related Outputs"
+      description="List all resources that were important to the creation of this output. Only published outputs on the CRN Hub will be available below."
```

Comment:
This looks different from figma. Maybe this text "List all ..." should not be part of this `LabeledMultiSelect`, maybe it should be a text above this `LabeledMultiSelect` component and here we wouldn't have a description.
<img width="1440" alt="Screenshot 2023-03-08 at 14 29 50" src="https://user-images.githubusercontent.com/16595804/223786604-05fb3546-4d68-4938-aa93-a414ceb66b63.png">